source/manage/statistics.rst

```diff
@@ -87,6 +87,12 @@ Master DB Conns
 Replica DB Conns
     The number of active connections currently on one or more of `your read replica databases <https://docs.mattermost.com/scale/high-availability-cluster.html#database-configuration>`__.
 
+Playbook count
```

Comment:
Not sure about this, but I think we used the copies "Total Playbooks" and "Total Playbook Runs". 

If you still want to reference as  xxx count, ignore my comment

Reply:
<img width="974" alt="screenshot-comm" src="https://user-images.githubusercontent.com/4096774/167796446-a79f3545-9e8a-4e4e-91e7-00758345e97e.png">